h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -682,6 +683,11 @@ public Frame scoreContributions(Frame frame, Key<Frame> destination_key, Job<Fra
             .outputFrame(destination_key, outputNames, null);
   }
 
+  @Override
+  public Frame scoreContributions(Frame frame, Key<Frame> destination_key, int topN, int topBottomN, boolean abs, Job<Frame> j) {
+    throw new NotImplementedException("Sorting of shap for XGBooost is not yet implemented");
```

Comment:
why not?

Reply:
I'll do it in scope of this task https://h2oai.atlassian.net/browse/PUBDEV-8084

Reply:
https://github.com/h2oai/h2o-3/pull/5491